src/components/terms.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+
+export default () => (
```

Comment:
Give this component name please. `export default const Terms = () => (` should do the trick.